features/0023-did-exchange/README.md

```diff
@@ -316,15 +317,17 @@ The exchange response message is used to complete the exchange. This message is
 }
 ```
 
-The key used in the signed attachment must be verified against the invitation's `recipientKeys` for continuity.
+The key used in the signed attachment must be verified against the responder’s DID document’s authentication relationships for continuity.
```

Comment:
The attachment is no longer being signed, correct?  So this needs to be changed?

Reply:
Text updated, let me know if it makes more sense now.